gateway-service-definitions/src/main/resources/services/yarnuiv2/3.0.0/service.xml

```diff
@@ -89,6 +89,14 @@
       <rewrite apply="YARNUIV2/yarnuiv2/inbound/config" to="request.url"/>
     </route>
 
+    <route path="/yarnuiv2/proxy/**">
+      <rewrite apply="YARNUIV2/yarnuiv2/outbound/headers/jobhistory/job" to="response.headers"/>
```

Comment:
This doesn't seem right? proxy path pointing to jobhistory?

Reply:
Also the proxy rewrite rule doesn't operate on headers like the jobhistory one does?

Reply:
It can be updated to response.body, but the rule will still be jobhistory one as that is what is defined in the rewrite.xml.

Reply:
@risdenk : Let me know if there is any other better way for this. I can't remove the rule, I can only change response.headers to response.body in case of proxy rule. But it is needed without which it fails.

Reply:
This still seems wrong. That applying /proxy/** is rewrting some jobhistory stuff.